### PR TITLE
🏗 Normalize RTV number before determining Brotli compressed size

### PR DESCRIPTION
**Pull request by danielrozenberg**
Attempt to fix the issue raised in https://github.com/ampproject/amphtml/pull/24418#issuecomment-529698706 based on suggestion in https://github.com/ampproject/amphtml/pull/24418#issuecomment-530151391

### CODE REVIEW

File: build-system/tasks/bundle-size.js [hunk 1]
```diff
@@ -70,8 +71,16 @@ function getGzippedBundleSize() {
  * @return {string} the bundle size in KB rounded to 2 decimal points.
  */
 function getBrotliBundleSize() {
+  // Brotli compressed size fluctuates because of changes in the RTV number, so
+  // normalize this across pull requests by replacing that RTV with a constant.
+  const normalizedFileContents = fs
+    .readFileSync(runtimeFile, 'utf8')
+    .replace(
+      new RegExp(`\\b${internalRuntimeVersion}\\b`, 'g'),
```
Comment (rsimha) [inline build-system/tasks/bundle-size.js]: I see you're using a whole word regex (with `\b`). Won't that exclude the RTV versions that are prefixed with `01`? I think we should drop the `\b` to include those as well.

Comment (jridgewell) [reply build-system/tasks/bundle-size.js]: Agreed.

Comment (danielrozenberg) [reply build-system/tasks/bundle-size.js]: You're right, I started trying to fix this by changing the regex from `\\b${internalRuntimeVersion}\\b` to `\\b(?:\d\d)?${internalRuntimeVersion}\\b`, but this seems like overkill. It's highly unlikely that in the real world this 13 digits string will randomly appear in the minified code and _not_ be representative of the RTV, wholly of partially. Fixed.

Comment (rsimha) [reply build-system/tasks/bundle-size.js]: Agree. They way you've done it now seems best.

**Pull request merged**